### ISSUE TITLE
‎A7AE is a capital letter not a lowercase letter.

### ISSUE THREAD

**Issue by moyogo** [googlei18n/noto-fonts#1026]
Noto Sans, Noto Serif and Noto Sans Mono (Version 2.000 from https://www.google.com/get/noto/ or https://github.com/googlei18n/noto-fonts-alpha) have the wrong shape for ‎U+A7AE.

U+A7AE Ɪ is the uppercase of U+026A ɪ. While U+026A ɪ is a x-height smallcap serifed i, its uppercase must have cap-height and be a serifed I (preferably wider or thicker serifs that look like top and bottom bars, similar to the top bar of the T, to differentiate it from the letter U+0049 I if it has serifs like in Noto Sans and Noto Sans Mono).

For example, here is IiꞮɪ <0049, 0069, A7AE, 026A>:
![notosans-regular-ii](https://user-images.githubusercontent.com/1923747/34648767-68b80456-f398-11e7-8505-82c66cb05b74.png)
In Noto Sans, A7AE Ɪ should be as tall as 0049 I, with wider serifs or top and bottom bars to help differentiating it from 0049 I since both are used in the same language orthography.

![notosansmono-regular-ii](https://user-images.githubusercontent.com/1923747/34648791-f0912420-f398-11e7-9a22-2144600ab2c3.png)
Same as Noto Sans.

![notoserif-regular-ii](https://user-images.githubusercontent.com/1923747/34648780-d36ad1d4-f398-11e7-90fc-540da453ee4c.png)
In Noto Serif, A7AE Ɪ should be as tall as 0049 I, its top and bottom bars are fine.

Compare with the *correct* shapes in Arial 6.98:
![arial-regular-ii](https://user-images.githubusercontent.com/1923747/34648820-8f888032-f399-11e7-8915-054617f3ff39.png)
or Ɪɪ <A7AE, 026A> and their actual smallcap in the work-in-progress Brill (to keep in mind for #913):
https://twitter.com/TiroTypeworks/status/807321889748041728

**Comment by verdy-p** [comment]
I also agree. What is misleading term is the name that was given to the letter pair, naming it a "Latin [capital/small] letter small capital i" where it should have been named "Latin [capital/small] letter headed i" or "Latin [capital/small] letter capped i" ; this letter is definitely not a "small capital" (and also not "serif" : the head of the letter is not a serif but the same horizontal top bar used in the Latin letter "t" or "T"; this head is also not necessarily like lowercase letters (the top bar can be higher than the x-heights, and reach the M-height, exactly like for "t" and "T")

another better name could have been "Latin [capital/small] large i": you see the concept, it's not based on actual height of the letter but on its width which contrasts clearly from the "normal" letter "i/I" by using horizontal strokes like in E,F,T,Z for capitals, or in t,z for lowercase

However your last image is correct for Arial, but Times is not correct for the lowercase letter using serifs, instead of using hat and foot bars that should be serifed too. This lowercase version in Times is a standard capital Latin letter in smallcap style (e.g. for small Roman numerals), and not the indended distinctive letter. Only Arial is fully correct, Times is only half-correct for the capital of the letter.

**Comment by moyogo** [comment]
> However your last image is correct for Arial, but Times is not correct for the lowercase letter using serifs, instead of using hat and foot bars that should be serifed too. This lowercase version in Times is a standard capital Latin letter in smallcap style (e.g. for small Roman numerals), and not the indended distinctive letter. Only Arial is fully correct, Times is only half-correct for the capital of the letter.

No. Times is correct when it comes to how this letter has been used and is used in IPA. The lowercase is a x-height smallcap with normal serifs in serif fonts.

**Comment by marekjez86** [comment]
test + test results with the current fonts
[en-1026.zip](https://github.com/googlefonts/noto-fonts/files/3292158/en-1026.zip)

**Comment by moyogo** [comment]
It doesn’t look like U+A7AE Ɪ has changed yet.

**Comment by moyogo** [comment]
In Noto Sans and Noto Sans Display, U+A7AE Ɪ has changed to be a capital i with serifs. This is better, but now it cannot be distinguished from capital i (at least not without using a feature that changes the shape of the capital i). It would be better if it had extended serifs like what is done in Noto Sans Mono.
<img width="205" alt="Screenshot 2020-07-02 at 20 07 59" src="https://user-images.githubusercontent.com/1923747/86399960-c3dbb800-bc9f-11ea-95da-d21c79c2ff45.png">
<img width="189" alt="image" src="https://user-images.githubusercontent.com/1923747/86400454-8e839a00-bca0-11ea-81cd-fc90075d5ffa.png">

The issue has been fixed in Noto Sans Mono, Noto Serif, Noto Serif Display.
<img width="264" alt="image" src="https://user-images.githubusercontent.com/1923747/86400310-495f6800-bca0-11ea-9c4c-f5c02e335011.png">
<img width="218" alt="image" src="https://user-images.githubusercontent.com/1923747/86400162-09988080-bca0-11ea-8644-cbbb65f63548.png">
<img width="211" alt="image" src="https://user-images.githubusercontent.com/1923747/86400383-6d22ae00-bca0-11ea-93df-807d6d8bd9f8.png">

**Comment by nizarsq** [comment]
@marekjez86 Any thoughts if NotoSans should distinguish between U+A7AEf and capital i?
<img width="602" alt="Screen Shot 2020-07-14 at 10 26 25 PM" src="https://user-images.githubusercontent.com/22158882/87506808-0fd41880-c621-11ea-89d0-7e658e7586fa.png">
